coforma/swift-tech-challenge
# Issue/data ingestion

# Title
Data ingestion for institutions as well as ADR and pre-commit updates for Python
Contributes to #20 

# Summary
This is a Python script that runs on a Lambda function to convert the CSV to a Dyanmo bucket. 

## Type of change


# How To Test
1. Check out the DynamoDb, verify some of the data is seeded
2. Very data matches from Dynamo tocsv
3. Add a file in the S3 bucket in the institutions folder and verify the trigger runs the lambda.  Not this file should be the first row of the CSV with the headers and any row of data that matches in format.

# Checklist:
